website/docs/r/eip.html.markdown

```diff
@@ -88,7 +88,7 @@ The following arguments are supported:
 
 * `vpc` - (Optional) Boolean if the EIP is in a VPC or not.
 * `instance` - (Optional) EC2 instance ID.
-* `network_interface` - (Optional) Network interface ID to associate with.
+* `network_interface` - (Optional) Network interface ID to associate with. Do not use this to associate the EIP to `aws_lb` or `aws_nat_gateway` resources. Instead use the `allocation_id` available in those resources to allow AWS to manage the association, otherwise you will see `AuthFailure` errors.
```

Comment:
IMO this belongs as info in a `~>` box

Reply:
@tombuildsstuff It shall be done! Updated -- please take a second look?